.github/workflows/python_build_and_publish.yml

```diff
@@ -0,0 +1,28 @@
+name: Python package build and publish
+
+on:
+  release:
+    types: [published]
```

Comment:
I'm not certain if ```created``` or ```published``` would be better here. The exact distinction is not well documented.